x-pack/plugins/siem/public/components/field_renderers/field_renderers.tsx

```diff
@@ -80,52 +79,51 @@ export const autonomousSystemRenderer = (
     getEmptyTagValue()
   );
 
-export const hostIdRenderer = (host: HostEcsFields, ipFilter?: string): React.ReactElement =>
-  host.id && host.ip && (!(ipFilter != null) || host.ip.includes(ipFilter)) ? (
-    <EuiFlexGroup alignItems="center" gutterSize="none">
-      <EuiFlexItem grow={false}>
-        {host.name && host.name[0] != null ? (
-          <DefaultDraggable id={`${IpOverviewId}-host-id`} field={'host.id'} value={host.id[0]}>
-            <HostDetailsLink hostName={host.name[0]}>{host.id}</HostDetailsLink>
-          </DefaultDraggable>
-        ) : (
-          <>{host.id}</>
-        )}
-      </EuiFlexItem>
-    </EuiFlexGroup>
+interface HostIdRendererTypes {
+  host: HostEcsFields;
+  ipFilter?: string;
+  noLink?: boolean;
+}
+
+export const hostIdRenderer = (data: HostIdRendererTypes): React.ReactElement =>
+  data.host.id &&
+  data.host.ip &&
+  (!(data.ipFilter != null) || data.host.ip.includes(data.ipFilter)) ? (
+    <>
+      {data.host.name && data.host.name[0] != null ? (
+        <DefaultDraggable id={`${IpOverviewId}-host-id`} field="host.id" value={data.host.id[0]}>
+          {data.noLink ? (
+            <>{data.host.id}</>
+          ) : (
+            <HostDetailsLink hostName={data.host.name[0]}>{data.host.id}</HostDetailsLink>
+          )}
+        </DefaultDraggable>
+      ) : (
+        <>{data.host.id}</>
+      )}
+    </>
   ) : (
     getEmptyTagValue()
   );
 
 export const hostNameRenderer = (host: HostEcsFields, ipFilter?: string): React.ReactElement =>
   host.name && host.name[0] && host.ip && (!(ipFilter != null) || host.ip.includes(ipFilter)) ? (
-    <EuiFlexGroup alignItems="center" gutterSize="none">
-      <EuiFlexItem grow={false}>
-        <DefaultDraggable id={`${IpOverviewId}-host-name`} field={'host.name'} value={host.name[0]}>
-          <HostDetailsLink hostName={host.name[0]}>
-            {host.name ? host.name : getEmptyTagValue()}
-          </HostDetailsLink>
-        </DefaultDraggable>
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <DefaultDraggable id={`${IpOverviewId}-host-name`} field={'host.name'} value={host.name[0]}>
+      <HostDetailsLink hostName={host.name[0]}>
+        {host.name ? host.name : getEmptyTagValue()}
+      </HostDetailsLink>
+    </DefaultDraggable>
   ) : (
     getEmptyTagValue()
   );
 
-export const whoisRenderer = (ip: string) => (
-  <>
-    <WhoIsLink domain={ip}>{i18n.VIEW_WHOIS}</WhoIsLink>
-    <ExternalLinkIcon />
-  </>
-);
+export const whoisRenderer = (ip: string) => <WhoIsLink domain={ip}>{i18n.VIEW_WHOIS}</WhoIsLink>;
 
 export const reputationRenderer = (ip: string): React.ReactElement => (
   <>
     <VirusTotalLink link={ip}>{i18n.VIEW_VIRUS_TOTAL}</VirusTotalLink>
-    <ExternalLinkIcon />
```

Comment:
For my own information, how does a user knows that it is an external link? by removing the icon.

Reply:
Because the URL domain is written out for the user, which I think is indication enough that it is an external link. I originally had it with the external link icon, but it looked too busy, redundant and unnecessary in this particular case.